lib/feedjira/parser/atom_entry.rb

```diff
@@ -23,10 +25,26 @@ class AtomEntry
       element :modified, :as => :updated
       elements :category, :as => :categories, :value => :term
       elements :link, :as => :links, :value => :href
+      elements :link, :as => :enclosures, :value => :href, :with => {:rel => "enclosure"}
+      elements :link, :as => :enclosure_types, :value => :type, :with => {:rel => "enclosure"}
+      elements :link, :as => :enclosure_lengths, :value => :type, :with => {:rel => "enclosure"}
 
       def url
         @url ||= links.first
       end
+
+      def enclosure_url
+        @enclosure_url ||= enclosures.first
+      end
+
+      def enclosure_type
+        @enclosure_type ||= enclosures.first
+      end
+
+      def enclosure_length
+        @enclosure_type ||= enclosures.first
+      end
```

Comment:
Looks to me all these methods return the same value.  Is that what you really mean?

Reply:
No, should be `@enclosure_url ||= enclosures.first&.url` etc (or equivalent if the safe operator `&.` isn't wanted)